tests/swagger_generation/test_marshmallow_to_swagger.py

```diff
@@ -189,6 +189,7 @@ class Foo(m.Schema):
 
     def test_required(self):
         class Foo(m.Schema):
+            b = m.fields.Integer(required=True)
```

Comment:
could add a `test_ordered` as well for that case

Reply:
Good idea.